docs/modules/data/protobuf.md

```diff
@@ -0,0 +1,361 @@
+# Protocol Documentation
```

Comment:
Is this generated? Would be great if it had a comment indicating that at least.

Reply:
Yes, it's generated. we could create our own md template though.